lib/src/button/Button.tsx

```diff
@@ -47,9 +47,7 @@ const DxcButton = ({
           disableRipple
           aria-disabled={disabled}
           tabIndex={disabled ? -1 : tabIndex}
-          onClick={() => {
-            onClick();
-          }}
+          onClick={onClick}
```

Comment:
These two things are not exactly the same. The old version doesn't send the event object to the handler, and the new one does.

I think we decided not to send anything to the onClick handler, do you remember this @GomezIvann??

Reply:
I see, I was trying to resolve the error showing when a button doesn't have onClick prop defined. The examples in the themeBuilder don't have it and locally it is crashing all the page.

Reply:
Yes, you are completely right, it was made on purpose. To solve this issue #953 we have to options: 

1. Give the prop a default value: `() => {}`.
2. Change the `onClick` event to mandatory, since I can't find out a scenario where it makes sense to not pass it 🤔 .

Reply:
Changed onClick to mandatory as I am also agree with the second point.

Reply:
I also suggest to not mark the prop as mandatory, just to avoid changes in the api of the component.

Reply:
Done as @GomezIvann suggested.